test/unit/override.spec.js

```diff
@@ -36,7 +36,7 @@ describe('@override', function () {
     (function () {
       class Child extends Parent {
         @override
-        speak(first, second) {}
+        speak (first, second) {}
```

Comment:
Personally, I agree with you.   The requirement for a space here is part of the eslint "semistandard" ruleset, and I'm not strongly advocating it....    As I said somewhere else, I picked that because it seemed a close match to existing code, but this whitespace was one change needed to conform to "semistandard".      

I think picking a canned standard is easier than maintaining our own ruleset, but I don't have broad knowledge of all the rulesets out there.